src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -79,6 +79,17 @@ export interface SavedObjectsBulkResponse<T = unknown> {
   saved_objects: Array<SavedObject<T>>;
 }
 
+/**
+ *
+ * @public
+ */
+export interface SavedObjectsFindResult<T = unknown> extends SavedObject<T> {
+  /**
+   * The ES search's `_score` of this result.
```

Comment:
```suggestion
   * The Elasticsearch `_score` for this result.
```

I think it's better to rather use the full name for Elasticsearch